aten/src/ATen/native/native_functions.yaml

```diff
@@ -5332,6 +5332,31 @@
     NestedTensorCPU, NestedTensorCUDA: NestedTensor_from_padded_and_nested_example
   autogen: _nested_from_padded_and_nested_example.out
 
+- func: _to_buffer(Tensor(a) self) -> Tensor(a)
+  variants: method
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_to_buffer
+
+- func: _to_buffer_copy(Tensor self) -> Tensor
+  variants: method
+  tags: view_copy
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_to_buffer_copy
+
+- func: _from_buffer_maybe_view(Tensor(a) self, Tensor reference) -> Tensor(a)
+  variants: method
+
+- func: _from_buffer(Tensor(a) self, Tensor reference) -> Tensor(a)
+  variants: method
```

Comment:
I think we want to start using `python_module: nested` for these btw.
That way, we can make them public API and just re-import them with docs in the `torch.nested` namespace!

Similar to what we do with linalg (cc @Lezcano are there any gotchas that we should know about? Any things we should be doing differently than linalg?)

Reply:
https://github.com/pytorch/pytorch/issues/83775